### PR TITLE
cgroupd: unblock SIGTERM to make stop_cgroupd actually work

### PR DESCRIPTION
**Pull request by Snorch**
Sometimes due to sigblockmask inheritance cgroupd can inherit SIGTERM blocked. That will lead cgroupd ignoring SIGTERM from stop_cgroupd() and CRIU will get stuck due to waiting for never-stopping cgroupd.

I see this happening in lxc-checkpoint, also saw this in OpenVZ jenkins on cgroup_inotify00 test.

### CODE REVIEW

File: criu/cgroup.c [hunk 1]
```diff
@@ -1955,6 +1969,9 @@ static int prepare_cgroup_sfd(CgroupEntry *ce)
  */
 static int cgroupd(int sk)
 {
+	if (cgroupd_unblock_sigterm())
```
Comment (avagin) [inline criu/cgroup.c:1972]: pls write a comment to explain why we need to unblock sigterm here

File: criu/cgroup.c [hunk 2]
```diff
@@ -1947,6 +1947,20 @@ static int prepare_cgroup_sfd(CgroupEntry *ce)
 	return 0;
 }
```
Comment (avagin) [inline criu/cgroup.c:1949]: static int cgroupd_unblock_sigterm(void)
{

**Comment by avagin** [comment]
LGTM

File: criu/cgroup.c (hunk 2 above)
Comment (Snorch) [reply criu/cgroup.c:1949]: Thanks for pointing this out, fixed.

**Pull request merged**